### PR TITLE
fix: add Vega-Lite as a dependency

### PR DESCRIPTION
**Pull request by kzhang31415**
Vega-Lite isn't installed despite being listed as a peer dependency of Vega-embed, installing Vega-lite as a dependency works as a fix for now.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,5 +38,8 @@
   },
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "dependencies": {
+    "vega-lite": "^5.18.1"
```
Comment (domoritz) [inline package.json]: Please add the dependency to the specific package that needs it.

**Review by domoritz — CHANGES_REQUESTED** [review]

**Comment by kzhang31415** [comment]
I've just moved the dependency to the vega-embed package since that's the one that needs it.

**Comment by domoritz** [comment]
I don't see the change to any package.json file.

**Comment by kzhang31415** [comment]
Sorry, it looks like my changes didn't pass some of the checks because I hadn't properly uninstalled my dependencies to test them.

**Comment by domoritz** [comment]
There are unrelated changes now. Please check your changes. 

I'll make this pull request as draft until you are confident that it's ready.

**Comment by kzhang31415** [comment]
Is there a way to test the checks on my local computer before sending it for approval? "npm ci" worked on my side but failed here for some reason.

**Comment by domoritz** [comment]
Npm run test

**Comment by kzhang31415** [comment]
Issue is gone now, seems to be fixed with the recent commit from sheer

**Pull request closed without merging**